src/main/java/biz/netcentric/maven/extension/repofromenv/FromEnvReposConfigurationProcessor.java

```diff
@@ -110,10 +112,23 @@ void configureMavenExecution(MavenExecutionRequest request, List<RepoFromEnv> re
             }
             
             request.addActiveProfile(PROFILE_ID_REPOSITORIES_FROM_ENV);
-
+            
+            if (!disableBypassMirrors) {
+                bypassMirrorsForRepositoryIds(request, reposFromEnv.stream().map(RepoFromEnv::getId).collect(Collectors.toList()));
+            }
```

Comment:
took a bit to get the head around this, so `MVN_DISABLE_BYPASS_MIRRORS` is there to disable the new "auto-disable" 👍